aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -59,6 +69,33 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
+template <typename input_t, typename output_t>
+void convert_indices_from_csr_to_coo_cpu(const Tensor& result, const Tensor& crow_indices, const Tensor& col_indices, const int64_t size) {
+  int64_t numel = crow_indices.numel();
+  const input_t* col_indices_data_in = col_indices.data_ptr<input_t>();
+  const input_t* crow_indices_data_in;
+  if (crow_indices.is_contiguous()) {
+    crow_indices_data_in = crow_indices.data_ptr<input_t>();
+  } else {
+    auto crow_indices_contiguous = crow_indices.contiguous();
+    crow_indices_data_in = crow_indices_contiguous.data_ptr<input_t>();
+  }
+  output_t* data_out = result.data_ptr<output_t>();
```

Comment:
There is no guarantee that the `result` Tensor is contiguous.

Reply:
Does it need to be? We're not doing anything with it here.

Reply:
You're saving the data to the memory blocks behind `data_out` by manipulating raw pointers, yes it has to be contiguous.
Since this is an internal function and the `out=` variant is not used it would be enough to just place `TORCH_INTERNAL_ASSERT(result.is_contiguous())` here.

Reply:
Got it.
Adding contiguous checks for `col_indices` too.